mongodb/mongo-init.js

```diff
@@ -0,0 +1,908 @@
+        ]
+    },
+    // 3 - TransactionClosePaymentRetryQueueConsumerV2 - CANCELLATION_REQUESTED
+    {
+        transactionId: "00000000000000000000000000000003",
+        status: 'CANCELLATION_REQUESTED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000003',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000003',
+                eventCode: 'TRANSACTION_USER_CANCELED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            }
+        ]
+    },
+    // 4 - TransactionClosePaymentRetryQueueConsumerV2 - CANCELLATION_REQUESTED - v2
+    {
+        transactionId: "00000000000000000000000000000004",
+        status: 'CANCELLATION_REQUESTED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000004',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000004',
+                eventCode: 'TRANSACTION_USER_CANCELED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            }
+        ]
+    },
+    // 5 - TransactionClosePaymentRetryQueueConsumerV2 - CLOSURE_REQUESTED
+    {
+        transactionId: "00000000000000000000000000000005",
+        status: 'CLOSURE_REQUESTED',
+        creationDate: "2025-01-13T09:18:16.500000000Z[Etc/UTC]",
+        events: [
+            {
+                eventId: '00000001-0000-0000-0000-000000000005',
+                eventCode: 'TRANSACTION_ACTIVATED_EVENT',
+                creationDate: "2025-01-13T09:18:16.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000002-0000-0000-0000-000000000005',
+                eventCode: 'TRANSACTION_AUTHORIZATION_REQUESTED_EVENT',
+                creationDate: "2025-01-13T09:18:17.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000003-0000-0000-0000-000000000005',
+                eventCode: 'TRANSACTION_AUTHORIZATION_COMPLETED_EVENT',
+                creationDate: "2025-01-13T09:18:18.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000004-0000-0000-0000-000000000005',
+                eventCode: 'TRANSACTION_CLOSURE_REQUESTED_EVENT',
+                creationDate: "2025-01-13T09:18:19.000000000Z[Etc/UTC]"
+            },
+            {
+                eventId: '00000005-0000-0000-0000-000000000005',
+                eventCode: 'TRANSACTION_CLOSURE_FAILED_EVENT',
+                creationDate: "2025-01-13T09:18:20.000000000Z[Etc/UTC]"
+            }
```

Comment:
```suggestion
            }
```

here we want a transaction in CLOSURE_REQUESTED state, right? so this event have to be removed